src/main/java/com/epam/rd/autotasks/springemployeecatalog/AppConstants.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.rd.autotasks.springemployeecatalog;
+
+public class AppConstants {
+    public static final String ALL_FROM_EMPLOYEE = "SELECT * FROM employee";
```

Comment:
Please avoid using * in sql queries, because it leads to nonoptimal sql execution